src/main/resources/templates/nowe.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"/>
+    <title>Dodawanie zadań</title>
+</head>
+<body>
+
+<div class="input-group-text">Dodaj nowe zadanie:</div><br>
+<form class="input-group input-group-sm mb-3" method="post" th:action="@{/add}">
+    <input type="text" th:field="*{task.name}" placeholder="nazwa zadania" required></br>
+    <input type="text" th:field="*{task.category}" placeholder="kategoria" required>
```

Comment:
Dużo lepiej gdyby było to pole wyboru

Reply:
Pewnie tak, poprzez foreach, jak poniżej?